datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -521,15 +521,16 @@ def process_metric(self, metric, scraper_config, metric_transformers=None):
             )
             return  # Ignore the metric
 
-        for ignore_pattern in scraper_config['_ignored_patterns']:
-            if fnmatchcase(metric.name, ignore_pattern):
-                # Metric must be ignored
-                # Cache the ignored metric name to avoid calling fnmatchcase in the next check run
-                scraper_config['_ignored_metrics'].add(metric.name)
-                self._send_telemetry_counter(
-                    self.TELEMETRY_COUNTER_METRICS_IGNORE_COUNT, len(metric.samples), scraper_config
-                )
-                return  # Ignore the metric
+        if scraper_config['_ignored_patterns']:
```

Comment:
nit: you can also wrap `if metric.name in scraper_config['_ignored_metrics']` if you change to `if config['ignore_metrics']`